src/config/config.c

```diff
@@ -1490,8 +1493,16 @@ void reread_config(void)
 		   strcmp(conf_copy.webserver.api.pwhash.v.s, config.webserver.api.pwhash.v.s) != 0)
 			delete_all_sessions();
 
+		// Check if privacy level was reduced. If so, we need to restart FTL
+		if(conf_copy.misc.privacylevel.v.privacy_level < config.misc.privacylevel.v.privacy_level)
+		{
+			log_info("Privacy level was reduced, restarting FTL");
```

Comment:
This part of the code (logging) is never reached

```
nanopi@nanopi:~$ cat /var/log/pihole/FTL.log | grep Privacy
nanopi@nanopi:~$ 
```

Reply:
It is:
```
2023-10-21 13:22:53.770 [1057441/T1057444] INFO: Privacy level was reduced, restarting FTL
```

Mind that there are two ways of changing FTL's configuration: *internally* (via the API) or *externally* (via manipulating `pihole.toml` by a third-party `pihole-FTL --config ...` call) and the code we are talking about here is in the *external* path. So if you run
```
pihole-FTL --config misc.privacylevel 0
```
after it had been set to, e.g., `1`, FTL will actually log this as seen above in my comment.

Reply:
I used the webinterface to change the privacy level